server/etcdmain/help.go

```diff
@@ -236,6 +236,8 @@ Experimental feature:
 	Warning is generated if requests take more than this duration.
   --experimental-txn-mode-write-with-shared-buffer 'true'
     Enable the write transaction to use a shared buffer in its readonly check operations.
+  --experimental-bootstrap-defrag-threshold-megabytes
```

Comment:
aligning with @mm4tt comment above, can we document what 0 means.

Reply:
Done.